ietf/templates/doc/document_html.html

```diff
@@ -1,11 +1,13 @@
 {# Copyright The IETF Trust 2016, All Rights Reserved #}
 <!DOCTYPE html>
+{% load analytical %}
```

Comment:
Good call, but what nudged you to notice to do this?

Reply:
I happened to look at `base.html`.